### PR TITLE
Added The Elvis Operator to Groovy docs - Issue #1181

### PR DESCRIPTION
**Pull request by Ribeiro**

### CODE REVIEW

File: groovy.html.markdown [hunk 1]
```diff
@@ -200,6 +200,14 @@ def y = 10
 def x = (y > 1) ? "worked" : "failed"
 assert x == "worked"
 
+//Groovy supports 'The Elvis Operator' too!
+//Instead of using the ternary operator:
+
+displayName = user.name ? user.name : 'Anonymous'
+
+//We can write like this:
```
Comment (vendethiel) [inline groovy.html.markdown]: write **it**

Comment (Ribeiro) [inline groovy.html.markdown]: Fixed

**Comment by vendethiel** [comment]
Thank you! :+1:

**Comment by Ribeiro** [comment]
You're welcome  =)

**Pull request merged**